std/container/rbtree.d

```diff
@@ -19,6 +19,43 @@ Authors: Steven Schveighoffer, $(WEB erdani.com, Andrei Alexandrescu)
 */
 module std.container.rbtree;
 
+///
+unittest
+{
+    import std.container: RedBlackTree;
+    import std.algorithm: equal;
+
+    auto rbt = new RedBlackTree!(int)(3, 1, 4, 2, 5);
```

Comment:
I think idiomatic D would be to use the factory method, which uses IFTI to infer the type.
```D
auto rbt = redBlackTree(3, 1, 4, 2, 5);
```